thirtyinch-kotlin/src/main/java/net/grandcentrix/thirtyinch/kotlin/TiPresenter.kt

```diff
@@ -0,0 +1,21 @@
+package net.grandcentrix.thirtyinch.kotlin
+
+import android.annotation.SuppressLint
+import net.grandcentrix.thirtyinch.TiPresenter
+import net.grandcentrix.thirtyinch.TiView
+
+/**
+ * Will call the given [block] in [TiPresenter.sendToView].
+ *
+ * This have the benefit that we can omit the `it` inside the `sendToView{}` call.
+ *
+ * Example:
+ * ```
+ * // Before
+ * presenter.sendToView { it.aViewMethod() }
+ * // After
+ * presenter.sendToViewKotlin { aViewMethod() }
+ * ```
+ */
+@SuppressLint("RestrictedApi")
+fun <V : TiView> TiPresenter<V>.sendToViewKotlin(block: V.() -> Unit) = sendToView { block.invoke(it) }
```

Comment:
How about at least name it `sendToViewKt` as all Kotlin stuff is suffixed with `Kt`?

Reply:
Thanks for the recommendation.
I think we should go with @Syex idea here because... well, it make sense 🙃 

Using a complete different name for an "extension" is just confusing IMO.

Reply:
I would prefer to go with @passsy 's suggestion. We should not include some language specific abbreviations in our public API.
My prefered candidate would be `withView`. Reason: It makes sense.